src/pot.erl

```diff
@@ -106,7 +106,7 @@ valid_totp(Token, Secret, Opts) ->
                 Token ->
                     true;
                 _ ->
-                    Window = proplists:get_value(window, Opts, 0),
+                    Window = proplists:get_value(addwindow, Opts, 0),
```

Comment:
This looks wrong to me. `window` is the parameter that **expands** the number of allowed interval tokens to be used (basically +/- `window` intervals are considered valid). `addwindow` is supposed to offset the interval that you check.

Reply:
Hi @nalundgaard, that's very useful to know! It would be great if the docs described these options in a bit more detail?

Reply:
@danturn Indeed! It's definitely worth tracking separately. The examples hint at what the options do, but they really don't explain in detail their intention, so you kind of have to piece it together from reading the code and being familiar with the hotp/totp RFC nomenclature.

Reply:
@danturn maybe you will like #22.